ranger-gcs-connector-adapter/src/main/java/com/google/cloud/hadoop/ranger/gcs/connectorAdapter/RangerProxyServiceAdapter.java

```diff
@@ -37,15 +37,19 @@
     private static final String CONNECTION_ERR_MSG =  "Can not connect to Ranger proxy server.";
 
     private String hostName;
-    private RangerAuthorizationProxyClient RangerRequestHandler;
+    private RangerRequestHandlerInterface rangerRequestHandlerInterface;
```

Comment:
don't name your variable with "interface" at the end. Just call it "rangerRequestHandler".